schema/sqlite/v3/temporal/schema.sql

```diff
@@ -0,0 +1,275 @@
+CREATE TABLE namespaces(
```

Comment:
@wxing1292 the path `schema/sqlite/v3/temporal/schema.sql` was from your patch -- any reason not to start at v0 or v1?

Reply:
driver name is `sqlite3`